server/setup.go

```diff
@@ -10,11 +10,19 @@ import (
 	"github.com/joshprzybyszewski/cribbage/server/persistence"
 	"github.com/joshprzybyszewski/cribbage/server/persistence/memory"
 	"github.com/joshprzybyszewski/cribbage/server/persistence/mongodb"
+	"github.com/joshprzybyszewski/cribbage/server/persistence/mysql"
 )
 
 var (
-	database = flag.String(`db`, `mongo`, `Set to the type of database to access. Options: "mongo", "memory"`)
+	database = flag.String(`db`, `mysql`, `Set to the type of database to access. Options: "mysql", "mongo", "memory"`)
 	dbURI    = flag.String(`dbURI`, ``, `The uri to the database. default empty string uses whatever localhost is`)
+
+	dsnUser     = flag.String(`dsn_user`, `root`, `The DSN user for the MySQL DB`)
+	dsnPassword = flag.String(`dsn_password`, ``, `The password for the user for the MySQL DB`)
+	dsnHost     = flag.String(`dsn_host`, `127.0.0.1`, `The host for the MySQL DB`)
+	dsnPort     = flag.Int(`dsn_port`, 3306, `The port for the MySQL DB`)
+	dsnParams   = flag.String(`dsn_params`, ``, `The params for the MySQL DB`)
+	mysqlDBName = flag.String(`mysql_db`, `cribbage`, `The name of the Database to connect to in mysql`)
```

Comment:
This is a lot of args to pass in. Maybe it would good to have the option to pull these from a YAML or JSON config file instead?

Reply:
I like the way you think. However, until we know more about how we choose to deploy this app, let's defer that decision until then and keep flags for now